### PR TITLE
Add hint on "flowchart" and "graph"

### PR DESCRIPTION
**Pull request by koppor**
## :bookmark_tabs: Summary

This PR changes two things:

### Add hint on `graph`

The documentation on flowcharts uses `flowchart`

image

When opening the live editor (https://mermaid.live/), it uses `graph` for a flow chart

image

The documentation contains the hint on `graph` only in "Other examples":

image

image

### Reformat notes

Currently, notes are formatted using `**Note:**`. According to https://vitepress.vuejs.org/guide/markdown#custom-containers, it is possible to have other stylings:

image

image

This PR uses that for the flowchart description.

## :straight_ruler: Design Decisions

- Consistency of tooling and documentation
- Using available styling syntax to provide a nicer rendering

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [x] :notebook: have added documentation (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by sidharthv96** [comment]
Graph has been replaced with Flowchart in https://github.com/mermaid-js/mermaid-live-editor/pull/654
Also, I just added the formatting rules into Contributing.md https://github.com/mermaid-js/mermaid/commit/4ed6ec1a4dd4d406f1bb28551fcca19f84bb19c4. Can you please update the PR?

**Comment by koppor** [comment]
Updated 😅

From https://github.com/mermaid-js/mermaid/commit/4ed6ec1a4dd4d406f1bb28551fcca19f84bb19c4 I see that <code>```note```</code> is converted to `> **Note:**<br>` and not `::: note<br>`. I assume, this is intended as you aim for a light highlight of that content?

**Comment by sidharthv96** [comment]
@koppor it converts based on the output format. Default output is github markdown, which uses the `**` syntax.
When building for vitepress, it'll convert to `:::` format.

File: packages/mermaid/src/docs/syntax/flowchart.md [hunk 1]
```diff
@@ -687,11 +697,13 @@ Starting with Mermaid version 9.4, you can use an alternate renderer named elk.
 The _elk_ renderer is an experimenal feature.
 You can change the renderer to elk by adding this directive:
 
-```
+```elk
```
Comment (sidharthv96) [inline packages/mermaid/src/docs/syntax/flowchart.md:700]: This is not elk language.

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

File: packages/mermaid/src/docs/syntax/flowchart.md (hunk 1 above)
Comment (koppor) [reply packages/mermaid/src/docs/syntax/flowchart.md]: I know, just rendered more nicely. Hack removed.

**Comment by koppor** [comment]
@sidharthv96 I resolved the conflicts again. Should I change something more?

**Comment by sidharthv96** [comment]
@koppor we have `note`, `tip`, `danger` and `warning`. 

`info` is not supported. `note` will be transformed to `info` when building the vitepress docs. So, don't worry.

**Comment by koppor** [comment]
@sidharthv96 Fixed.

**Pull request merged**